test/functional/mempool_persist.py

```diff
@@ -85,5 +91,29 @@ def run_test(self):
         self.nodes.append(self.start_node(0, self.options.tmpdir))
         wait_until(lambda: len(self.nodes[0].getrawmempool()) == 5)
 
+        regtestdir0 = os.path.join(self.options.tmpdir, 'node0', 'regtest')
```

Comment:
Intermediate `regtestdir0` variable not required. `os.path.join()` can take any number of arguments.

Same for `regtestdir1` below.

Reply:
Makes sense. Will submit a fix for it